cli/forest_utils.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python3.9
+# Copyright (c) 2021 MobileCoin Inc.
+# Copyright (c) 2021 The Forest Team
+import functools
+import logging
+import os
+from typing import Optional, cast, Dict
+
+
+def FuckAiohttp(record: logging.LogRecord) -> bool:
```

Comment:
we should probably rename this to something else

Reply:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/77425619/195652178-5c9dee17-1533-4021-af66-9a33affaf141.png">

fine, fine